packages/roosterjs-content-model-core/lib/corePlugin/selection/SelectionPlugin.ts

```diff
@@ -219,11 +219,13 @@ class SelectionPlugin implements PluginWithState<SelectionPluginState> {
                 }
             }
 
-            this.state.mouseDisposer = editor.attachDomEvent({
-                mousemove: {
-                    beforeDispatch: this.onMouseMove,
-                },
-            });
+            if (!this.state.mouseDisposer) {
+                this.state.mouseDisposer = editor.attachDomEvent({
```

Comment:
how about we just dispose the previous one and then attach a new one? I'm afraid if in a corner case this may cause the event can never be attached.

Reply:
That works for me, changed in latest commit.